### DISCUSSION TITLE
查询效率起伏不定问题

### DISCUSSION THREAD

**Discussion by young138120** [apache/doris#10536] [A - General / Q&A] [answered]
通过spark connector将批量数据插入到doris中，大概三十几万数据  
插入数据后，马上进行查询，查询时间很长，大概15秒左右  
过了大概半小时后，查询效率就很高，大概几十毫秒  

再查询很高的那段时间，查了集群状态没有异常  
想知道这是为什么，从哪方面能够知悉细节

**Comment by hf200012** [comment, marked as answer]
这个是因为数据入doris之后，有一个数据compaction该过程，compaction完成之后数据查询效率更高

**Reply by young138120** [reply]
我猜想也是这个原因，但是我想知道，我怎么观测compaction的这个过程呢
例如进度，效率相关的比较重要的信息

**Reply by dataalive** [reply]
现在可以通过top -H看到这个线程的情况，但是没办法更直接的去看某一批数据的compaction情况。

**Reply by young138120** [reply]
> 现在可以通过top -H看到这个线程的情况，但是没办法更直接的去看某一批数据的compaction情况。

后续有计划做这块的偏监控的功能吗？
